Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -647,8 +647,11 @@ WebPage::WebPage(PageIdentifier pageID, WebPageCreationParameters&& parameters)
         ProcessCapabilities::setHardwareAcceleratedDecodingDisabled(true);
         ProcessCapabilities::setCanUseAcceleratedBuffers(false);
 #if HAVE(CGIMAGESOURCE_DISABLE_HARDWARE_DECODING)
-        OSStatus ok = CGImageSourceDisableHardwareDecoding();
-        ASSERT_UNUSED(ok, ok == noErr);
+        static std::once_flag onceFlag;
+        std::call_once(onceFlag, [] {
```

Comment:
This is single-threaded code, std::call_once() introduces locking for no good reason here. A static boolean flag would have sufficed here.